pkg/cmd/pipeline/start.go

```diff
@@ -395,6 +395,11 @@ func (opt *startOptions) startPipeline(pName string) error {
 		if err != nil {
 			return err
 		}
+		if len(prLast.ObjectMeta.GenerateName) > 0 {
```

Comment:
Could you add a unit test for if `len(prLast.ObjectMeta.GenerateName) > 0`?

Reply:
okay, I'll consider it!!

Reply:
okay, I'll consider it!!